bundles/AdminBundle/Resources/public/js/pimcore/object/helpers/import/configDialog.js

```diff
@@ -773,5 +795,17 @@ pimcore.object.helpers.import.configDialog = Class.create({
                 pimcore.helpers.showNotification(t("error"), message, "error");
             }.bind(this)
         });
+    },
+    
+    getExportConfigUrl: function(){
+        this.commitEverything();
+        var config = this.prepareSaveData();
+        config = Ext.encode(config);
+        
+        return "/admin/object-helper/export-json-config?classId="+this.classId+"&config="+config;
```

Comment:
same similar to import url

Reply:
Done, renamed as "export-csv-import-config-as-json"